docs/src/components/landing-page/TwitterShowcaseCards.vue

```diff
@@ -1,8 +1,6 @@
 <template>
   <q-carousel
     v-model="slide"
-    transition-prev="scale"
```

Comment:
We need to find a way to pre-load the next tweets page when showing the current one, as otherwise there's that loading moment without any indication which isn't really nice. Or add a cool loading animation, but that's possibly even more complex, I'll probably need to look into this myself